lib/base/scriptglobal.cpp

```diff
@@ -59,7 +59,7 @@ void ScriptGlobal::Set(const String& name, const Value& value, bool overrideFroz
 			}
 		}
 
-		parent->SetFieldByName(tokens[tokens.size() - 1], value, overrideFrozen, DebugInfo());
+		parent->SetFieldByName(tokens[tokens.size() - 1], value, false, DebugInfo());
```

Comment:
As `parent` is a `Namespace::Ptr`, I'd just directly use `Set()` here which doesn't even have parameters that are then ignored (bonus: also has the `DebugInfo()` default value).